sympy/matrices/eigen.py

```diff
@@ -557,7 +557,7 @@ def _bidiagonal_decomposition(M, upper=True):
 
 def _bidiagonalize(M, upper=True):
     """
-    Returns `B`
+    Returns ``B``
 
     where B is the Bidiagonalized form of the input matrix.
```

Comment:
The blank line in the middle of this sentence should be removed, and the two `B`s should either both be math or code.